game/src/engine/config/ConfigService.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <filesystem>
+
+#include <yaml-cpp/yaml.h>
+#include "engine/service/Service.h"
+#include "VehicleData.h"
+
+namespace fs = std::filesystem;
```

Comment:
It's usually better to avoid adding any sort of aliases (`using ...` or `namespace x = ...`) at the top-level header scope if only doing it to make the name shorter to type because then anything including `ConfigService.h` will also have this alias/define, and can sometimes be confusing if the namespace `fs` magically appears 3-4 includes up

In the CPP file feel free to alias as much as you want tho!